src/lib/common/utils.ts

```diff
@@ -0,0 +1,11 @@
+interface WithLabels {
+  labels: string[];
+}
+
+export function discernTags(posts: WithLabels[]): string[] {
```

Comment:
Let's create a couple test cases ([_example_](https://github.com/EthanThatOneKid/acmcsuf.com/blob/dcd14855e9e14b402b2c72d41f16a0b26d5f7bef/src/lib/common/utils.test.ts#L4)) in `/src/lib/common/utils.test.ts` for `discernTags()` to make sure we have the behavior we want.